frame/collective/src/lib.rs

```diff
@@ -214,9 +217,9 @@ mod weight_for {
 		proposal: impl Into<Weight>,
 	) -> Weight {
 		db.reads(1) // read members for `is_member`
-			+ 86_000_000 // constant
-			+ 5_000 * MAX_ASSUMED_PROPOSAL_BYTES // B
-			+ 330_000 * members.into() // M
+			+ 23_000_000 // constant
+			+ 3_000 * MAX_ASSUMED_PROPOSAL_BYTES // B
+			+ 120_000 * members.into() // M
 			.saturating_add(proposal.into()) // P
```

Comment:
This operation is executed as: `120_000 * ( members.into().saturating_add(proposal.into()) )`

Reply:
oh damn, nice catch(es)